flocker/node/_deploy.py

```diff
@@ -135,6 +129,8 @@ def __init__(self, node_uuid, local_applications, leases):
         :param applications: Applications running on the node.
         :param Leases leases: The current leases on datasets.
         """
+        if local_applications is None:
+            local_applications = []
```

Comment:
You might have told me before, in which case I'm sorry for asking again. How is this change related to making container API optional?